docker/Dockerfile

```diff
@@ -36,17 +36,14 @@ ARG BAZEL_VERSION=5.2.0
 RUN wget -q https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION}/bazel-${BAZEL_VERSION}-linux-x86_64 -O /usr/bin/bazel \
  && chmod a+x /usr/bin/bazel
 
-# Remove cuda symlink to avoid bazel circle symlink errors
-RUN rm /usr/local/cuda-11.7/cuda-11.7
-
```

Comment:
Removed cuda symlink deletion, as build fails with this line on Ubuntu 20.04 since file does not exist